front-end/components/SettingsPanel/panels/Personalize/personalize.vue

```diff
@@ -5,11 +5,67 @@
             To increase the size of the dashboard, please use the built in zoom functionality using `ctrl +` and `ctrl
             -`.
         </p>
+        <p><strong>Hide cursor:</strong></p>
+        <button class="option" @click="disableCursorHide" :class="{ current: !cursorHidden }">Don't hide cursor</button>
+        <button class="option" @click="enableCursorHide" :class="{ current: cursorHidden }">Hide cursor after:</button>
```

Comment:
```suggestion
        <button class="option" @click="enableCursorHide" :class="{ current: cursorHidden }">Hide cursor</button>
```
I looks a bit weird in the settings panel when you have `hide cursor after:` and than comes nothing.